server_addon/create_ayon_addons.py

```diff
@@ -268,22 +272,37 @@ def create_addon_package(
         )
 
 
-def main(create_zip=True, keep_source=False):
+def main(
+    output_dir=None,
+    skip_zip=True,
+    keep_source=False,
+    clear_output_dir=False,
+    addons=None,
+):
     current_dir = Path(os.path.dirname(os.path.abspath(__file__)))
     root_dir = current_dir.parent
-    output_dir = current_dir / "packages"
-    print("Package creation started...")
+    create_zip = not skip_zip
 
-    # Make sure package dir is empty
-    if output_dir.exists():
+    if output_dir:
+        output_dir = Path(output_dir)
+    else:
+        output_dir = current_dir / "packages"
+
+    if output_dir.exists() and clear_output_dir:
         shutil.rmtree(str(output_dir))
```

Comment:
Would be nice if this would also **print** that it's purging/removing the output directory - just so the user is aware from the output logs of the command?